main.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+
+    <link rel="stylesheet" type="text/css" href="main.css" />
+    <script type="text/javascript" src="data.js"></script>
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/markdown-it/8.4.2/markdown-it.js"></script>
+    <link href="https://fonts.googleapis.com/css?family=Merriweather" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Pacifico" rel="stylesheet">
+    
+</head>
+<body>
+
+<header class="main-header">
+
+
+        <div class="title-container">
+       <div class="page-title"><h1 >A Seattle Coder's Learning Blog</h1></div>
+        <div class="page-subtitle"><h3>A Woman in Tech, Breaking Stereotypes, and Broadening Inclusion.</h3></div>
+    </div>
+        <nav class="main-nav">
+            <a class="nav-link" href="#home-page">Home</a>
+            <a class="nav-link" href="#about-page">About</a>
+            <a class="nav-link" href="#all-posts-page">AllPosts</a>
+            <a class="nav-link"href="#contact-page">Contact</a>
+        </nav>
+
+    </header>
+
+   
+<div class="all-posts-page">
+    <div id="all-posts-container">
+        <div class="post-container"> 
```

Comment:
this post container is the template and is rendering as an empty `<div>` in the browser. it should be commented out, or removed if you don't need to reference it any more